EvidenceApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -57,13 +75,17 @@ private static string GetTemplateIdFor(DeliveryMethod deliveryMethod, Communicat
                 {
                     CommunicationReason.EvidenceRequest => Environment.GetEnvironmentVariable(
                         "NOTIFY_TEMPLATE_EVIDENCE_REQUESTED_SMS"),
+                    CommunicationReason.EvidenceRejected => Environment.GetEnvironmentVariable(
+                        "NOTIFY_TEMPLATE_EVIDENCE_REJECTED_SMS"),
                     _ => throw new ArgumentOutOfRangeException(nameof(communicationReason), communicationReason,
                         $"Communication Reason {communicationReason.ToString()} not recognised")
                 },
                 DeliveryMethod.Email => communicationReason switch
                 {
                     CommunicationReason.EvidenceRequest => Environment.GetEnvironmentVariable(
                         "NOTIFY_TEMPLATE_EVIDENCE_REQUESTED_EMAIL"),
+                    CommunicationReason.EvidenceRejected => Environment.GetEnvironmentVariable(
+                        "NOTIFY_TEMPLATE_EVIDENCE_REJECTED_EMAIL"),
```

Comment:
I think you'll need to set this up in `serverless.yml` see [here](https://github.com/LBHackney-IT/evidence-api/blob/0143372649e0d877add7a0e5dcbce1b11c20d3de/EvidenceApi/serverless.yml#L30)